src/accessors/IPersistenceRead.ts

```diff
@@ -4,7 +4,7 @@ import { IRelatedPersistenceRead } from './IRelatedPersistenceRead';
  * Provides access to the database
  */
 
-export interface IPersistence {
+export interface IPersistenceRead {
```

Comment:
It was a slip up in your pull request, was easier to merge as is and update my version.